tt-train/tests/ops/unary_ops_test.cpp

```diff
@@ -67,3 +68,145 @@ TEST_F(UnaryOpsTest, LogSoftmax) {
         EXPECT_NEAR(tensor_grad[idx], expected_grad[idx], 2e-2F);
     }
 }
+
+TEST_F(UnaryOpsTest, Silu) {
+    namespace core = ttml::core;
```

Comment:
you can define a whole test inside of the namespace ttml::core, I think it will be the best case.
just make namespace ttml::core::tests. It will be our new standard